wallet/how-to/connect/set-up-sdk/gaming/unity.md

```diff
@@ -163,83 +188,6 @@ Create a new configuration by right-clicking on the project window and navigatin
 **MetaMask > Config**.
 Name the new configuration and use it when initializing the `MetaMaskUnity` instance.
 
-## API
-
-The following is an overview of the APIs from the most important classes.
-
-### MetaMaskUnity
-
-This is a singleton class that you can use to access the `MetaMaskWallet` instance, which is
-specific to Unity.
-
-#### `Instance`
-
-This is the singleton instance of the `MetaMaskUnity` class that is lazy-loaded when you access it
-for the first time.
-
-#### `Initialize`
-
-This method initializes the `MetaMaskWallet` instance and makes it accessible via the `Wallet` property.
-You can also pass extra options and parameters to it to customize the wallet instance:
-
-```csharp
-// Initialize using default settings
-MetaMaskUnity.Instance.Initialize();
-
-// Initialize using custom transport and socket provider
-var transport = new MyCustomTransport();
-var socketProvider = new MyCustomSocketProvider();
-MetaMaskUnity.Instance.Initialize(transport, socketProvider);
-
-// Initialize using custom config, transport and socket provider
-var config = myMetaMaskConfig;
-var transport = new MyCustomTransport();
-var socketProvider = new MyCustomSocketProvider();
-MetaMaskUnity.Instance.Initialize(config, transport, socketProvider);
-```
-
-#### `SaveSession`
-
-This method saves the current session to the persistent storage.
-This is useful when you want to save the session and restore it later.
-This is automatically called when the application quits, but you can also manually call it.
-
-#### `LoadSession`
-
-This method loads the session from the persistent storage.
-This is useful when you want to restore the session after the application quits.
-This is automatically called when the application starts, but you can also manually call it.
```

Comment:
Maybe... "When the application starts, it automatically calls this method. You can also manually call this method."

Hmm... A little wordy to get rid of the passive voice.